spec/builtin-history.test.sh

```diff
@@ -32,29 +34,65 @@ history -a
 ^D
 ## END
 
-#### history -r
-rm -f tmp
-echo 'foo' > tmp
 
-echo '
+#### history -w
+
+cd $TMP
+echo 'cmd orig' > tmp
+HISTFILE=tmp
 history -c
+echo 'cmd new' > /dev/null
+history -w
+grep 'orig' tmp > /dev/null
+echo "found=$?"
+## STDOUT:
+found=1
+## END
 
+
+#### history -r 
```

Comment:
I think it will help to move the comment up to the description line, so that we can see it in the console / HTML reports

e.g.

"history -r reads from history file and appends to history in memory"